website/projects/scimaps/src/app/news-item/news-item.component.html

```diff
@@ -0,0 +1,8 @@
+<a class="news-item" [href]="item.pdfLink">
```

Comment:
Merge develop and remove this. It is in the wrong place and develop has it correctly